pycid/core/macid.py

```diff
@@ -51,20 +50,14 @@ def get_ne(self, solver: Optional[str] = "enumpure") -> List[List[StochasticFunc
         """
         return self.get_ne_in_sg(solver=solver)
 
-    def joint_pure_policies(self, decisions: Iterable[str]) -> List[Tuple[StochasticFunctionCPD, ...]]:
-        """return a list of tuples of all joint pure policies in the MACID. A joint pure policy assigns a
-        pure decision rule to every decision node in the MACID."""
-        all_dec_decision_rules = list(map(self.pure_decision_rules, decisions))
-        return list(itertools.product(*all_dec_decision_rules))
-
     def get_ne_in_sg(
         self,
         decisions_in_sg: Optional[Iterable[str]] = None,
         solver: Optional[str] = "enumpure",
     ) -> List[List[StochasticFunctionCPD]]:
         """
         Return a list of NE in a MACID subgame. By default, this finds all pure NE in an arbitray N-player game.
```

Comment:
like above, I'm not sure about this being the default behaviour. In a 2 player MACID, I would want get_ne() to return all NE